apps/stack/template.yaml

```diff
@@ -220,7 +220,21 @@ Conditions:
     - !Equals
       - !Ref DatasourceID
       - ""
-
+  HasLogGroupNamePatterns: !Not
```

Comment:
Is there a * sentinel value that's valid for this?

Reply:
You can enter * for `LogGroupNamePatterns` and it will collect everything, yes